rice-middleware/web/src/main/java/javax/rmi/PortableRemoteObject.java

```diff
@@ -0,0 +1,42 @@
+package javax.rmi;
+
+import java.rmi.NoSuchObjectException;
+import java.rmi.Remote;
+import java.rmi.RemoteException;
+
+/**
+ * This is being provided as a no-op shim since JOTM depends on this class but
+ * it does nothing with it.  This class was removed from the JDK for Java 11
+ * when CORBA was removed.
+ */
```

Comment:
This comment mostly explains what this class is used for.  It is provided because I wasn't finding a replacement JAR which satisfied JOTM's needs.